packages/contracts/contracts/optimistic-ethereum/ovm/StateManager.sol

```diff
@@ -20,6 +20,7 @@ contract StateManager {
     // Contract code storage / contract address retrieval
     function associateCodeContract(address _ovmContractAddress, address _codeContractAddress) public;
     function getCodeContractAddress(address _ovmContractAddress) external view returns(address);
+    function getOvmContractAddress(address _codeContractAddress) external view returns(address);
```

Comment:
Same thing here -- would name it `getOvmContractAddressFromContractAddress(...)` (and the same for the other one) so where you call it, it's clear what the parameter is without having to open up the contract and look.